clients/cloud/python/consumer_ccsr.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Consume messages from Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+# Reads Avro data, integration with Confluent Cloud Schema Registry
+#
+# =============================================================================
+
+from confluent_kafka import Consumer
+from confluent_kafka.avro import AvroConsumer
+from confluent_kafka.avro.serializer import SerializerError
+import json
+import ccloud_lib
+
+
+if __name__ == '__main__':
+
+    # Initialization
+    args = ccloud_lib.parse_args()
+    config_file = args.config_file
+    topic = args.topic
+    conf = ccloud_lib.read_ccloud_config(config_file)
+
+    # Create Avro Consumer instance
+    # 'auto.offset.reset=earliest' to start reading from the beginning of the
+    #   topic if no committed offsets exist
+    c = AvroConsumer({
+        'bootstrap.servers': conf['bootstrap.servers'],
+        'sasl.mechanisms': 'PLAIN',
+        'security.protocol': 'SASL_SSL',
+        'sasl.username': conf['sasl.username'],
+        'sasl.password': conf['sasl.password'],
+        'schema.registry.url': conf['schema.registry.url'],
+        'schema.registry.basic.auth.credentials.source': conf['basic.auth.credentials.source'],
+        'schema.registry.basic.auth.user.info': conf['schema.registry.basic.auth.user.info'],
```

Comment:
Are SR cloud credentials different from your standard ccloud credentials? If not it might be worth showing an example of the `SASL_INHERIT` provider which would be much more convenient.

Reply:
They are different